### PR TITLE
[DOCFIX] Add Minio configuration details

### PR DESCRIPTION
**Pull request by NitishT**

### CODE REVIEW

**Comment by NitishT** [comment]
@alluxio-bot , check this please

**Comment by gpang** [comment]
ok to test

**Comment by gpang** [comment]
Jenkins, add to whitelist

File: docs/en/Configuring-Alluxio-with-Minio.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+priority: 0
+---
+
+* Table of Contents
+{:toc}
+
+This guide describes how to configure Alluxio with [Minio](https://minio.io/) as the
+under storage system. Alluxio natively provides two different client implementations for accessing
+Minio, aws-sdk-java-s3 through the s3a:// scheme (recommended for better performance) and jets3t
+through the s3n:// scheme.
```
Comment (calvinjia) [inline docs/en/Configuring-Alluxio-with-Minio.md]: I don't think Minio is compatible through `s3n` right?

Comment (NitishT) [inline docs/en/Configuring-Alluxio-with-Minio.md]: That's true @calvinjia I removed the reference to `s3n`

**Comment by NitishT** [comment]
@calvinjia anything else pending for this from my side?

**Comment by calvinjia** [comment]
@NitishT Thanks for adding these instructions!

**Comment by NitishT** [comment]
Thanks for your patience and review @calvinjia

**Comment by NitishT** [comment]
Hey @calvinjia I could not find Minio under the `Under Stores` section on Alluxio docs site: http://www.alluxio.org/docs/master/en/ Is there something else I need to do - any other files to be updated?

**Comment by calvinjia** [comment]
@NitishT You don't need to do anything else, the Alluxio site will be automatically pick up the latest changes to master when it gets updated!

**Comment by NitishT** [comment]
Understood. Thanks @calvinjia !

**Pull request merged**